portal/config.py

```diff
@@ -105,6 +105,9 @@ class BaseConfig(object):
 
     SYSTEM_TYPE = 'development'
 
+    SMARTLING_LANGUAGES = ['en_AU', 'es_MX', 'fr_CA']
+    SMARTLING_PROJECT_ID = '13f8e2dcf'
```

Comment:
Shouldn't the project be unique for TrueNTH versus ePROMs?  If so, please move this value to the respective site_persistence files.  Probably true for the languages as well?